fenics_ice/inout.py

```diff
@@ -306,37 +306,85 @@ def field_from_vel_file(infile, field_name):
 
 def read_vel_obs(params, model=None):
     """
-    Read velocity observations & uncertainty from HDF5 file
-
-    For now, expects an HDF5 file
+    Reads velocity observations & uncertainty from a HDF5 file
+    containing:
+    - point cloud velocities (u_obs, v_obs) for inversion only
+    - gridded data set of composite ice velocities & uncertainties
+    use for alpha initialisation and boundary conditions
+    All variables in the HDF5 file should be with the form
+    (values, )
+
+    Generates self.u_obs, self.v_obs, self.u_std, self.v_std,
+    self.uv_obs_pts, self.mask_vel
+    self.u_comp, self.v_comp, self.u_comp_std, self.v_comp_std,
+    self.uv_comp_pts
     """
     infile = Path(params.io.input_dir) / params.obs.vel_file
     assert infile.exists(), f"Couldn't find velocity observations file: {infile}"
 
     infile = h5py.File(infile, 'r')
+    keys = list(infile.keys())
+
+    assert 'u_obs' in keys, \
+        f"Your vel file has not been configure correctly " \
+        f"make sure you have velocity data with the correct var names " \
+        f"x, y, u_obs, v_obs, u_obs_std, v_obs_std "
```

Comment:
Assert not needed -- only checks for one key, and the following code should raise an error if any are missing.

Reply:
deleted see [here](https://github.com/EdiGlacUQ/fenics_ice/pull/13/commits/f07ec9a485c21fe3546b1e72f7c61733694aa11b#diff-958ce5f68bfa9888398edc3c3324d7c91c20ed48e63e69aa85f418fc1fd64bfdL328-L331)